raven.lua

```diff
@@ -484,7 +485,7 @@ function _M.gen_capture_err(self)
 end
 
 -- HTTP request template
-local xsentryauth_http = "POST %s HTTP/1.0\r\nHost: %s\r\nConnection: close\r\nContent-Type: application/json\r\nContent-Length: %d\r\nUser-Agent: %s\r\nX-Sentry-Auth: Sentry sentry_version=5, sentry_client=%s, sentry_timestamp=%s, sentry_key=%s, sentry_secret=%s\r\n\r\n%s"
+local xsentryauth_http = "POST %s HTTP/1.0\r\nHost: %s\r\nConnection: close\r\nContent-Type: application/json\r\nContent-Length: %d\r\nUser-Agent: %s\r\nX-Sentry-Auth: Sentry sentry_version=7, sentry_client=%s, sentry_timestamp=%s, sentry_key=%s, sentry_secret=%s\r\n\r\n%s"
```

Comment:
`sentry_version` is the expected Sentry server version that raven client talks to. At the time when I submitted my PR six month ago, the Sentry server version was at 7.x, now it's at 8.x. So IMMO it should in fact be 8 now, pending testing with latest 8.x version. (I did test again Sentry 7.x server back then.) However, I don't think that matters as the unit tests are using different versions all over the place in both server side and client side codebase.

I've open a ticket https://github.com/getsentry/raven-python/issues/908 to ask Sentry guys to clarify this. I don't the `sentry_version` actually matters if you are putting it as 5 or 6 or 7 or 8. But if you insist, I can make it to 6.